readme.md

```diff
@@ -93,6 +93,17 @@ alternatively make a debian package with:
 
 this puts the built packages in `../`
 
+
+#### Static Linux
+
+static native optimized:
+
+    $ make static STATIC_LINK=ON DOWNLOAD_SODIUM=ON 
```

Comment:
Does this work if I have libsodium-dev installed?  I thought that DOWNLOAD_SODIUM is more of an "it's okay to download if you need to" option.  Perhaps we need some sort of `DOWNLOAD_SODIUM=ALWAYS` setting for it?

Reply:
libsodium will be statically linked if installed the `DOWNLOAD_SODIUM` part is probably not needed in this case.